playbooks/ci-ceph_swift/templates/usr/bin/lshw

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+hostname="{{ inventory_hostname }}"
+
+{% raw -%}
+# run the original lshw command
+if [ "$1" != "-C" -o "$2" != "disk" ]; then
+  if [ -f /usr/bin/lshw.original ]; then
+    sudo /usr/bin/lshw.original
```

Comment:
Yeah, these are very good points! I have refined that part as follows which with your suggestions applied and also passing all parameters to the original lshw script to avoid any gap in the final output:
```
# run the original lshw command
if [ "$1" != "-C" -o "$2" != "disk" ]; then
  if [ -f /usr/bin/lshw.original ]; then
    /usr/bin/lshw.original $@
    exit 0
  fi
fi
```